js/acquia_lift.debugger.js

```diff
@@ -0,0 +1,1885 @@
+/**
+ * Prevents parent element from scrolling when hovering over debugger elements.
+ */
+
+function preventDefault(e) {
```

Comment:
Can this be renamed to something that isn't already a common function name?  Maybe "preventScrollDefault".  When I read it quickly it's too close to e.preventDefault().